scene/resources/packed_scene.h

```diff
@@ -174,6 +184,12 @@ class SceneState : public RefCounted {
 	Vector<StringName> get_node_groups(int p_idx) const;
 	int get_node_index(int p_idx) const;
 
+	int32_t get_scene_local_id_offset() const;
+	void set_scene_local_id_offset(int32_t p_value);
+	int32_t get_node_local_id(int p_idx) const;
```

Comment:
In xml docs the return value is specified as `int`, but it's using `int32_t` - it *should* be the same on most platforms, but the definition of `int` is so vaque I think I'd use the same type in both places.